pkg/cedar/core/software_products.go

```diff
@@ -0,0 +1,85 @@
+package cedarcore
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/guregu/null"
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+
+	software_products "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/software_products"
+)
+
+// GetSoftwareProductsBySystem queries CEDAR for software product/tooling information associated with a particular system, taking the version-independent ID of a system
+func (c *Client) GetSoftwareProductsBySystem(ctx context.Context, cedarSystemID string) (*models.CedarSoftwareProducts, error) {
+	if !c.cedarCoreEnabled(ctx) {
+		appcontext.ZLogger(ctx).Info("CEDAR Core is disabled")
+		return nil, nil
```

Comment:
It might be good to return a `&models.CedarSoftwareProducts{}` here, but if not we can add that later along with mock data for local testing.